a/activemq/Dockerfiles/latest_centos_7/build/build.sh

```diff
@@ -0,0 +1,7 @@
+git clone https://github.com/apache/activemq 
```

Comment:
we generally keep the build scripts in the top level package folder which would be https://github.com/ppc64le/build-scripts/tree/master/a/activemq in this case and also have a specific header for each script like https://github.com/ppc64le/build-scripts/blob/master/a/aaargh/aaargh_rhel_7.3.sh#L1 to https://github.com/ppc64le/build-scripts/blob/master/a/aaargh/aaargh_rhel_7.3.sh#L16 which lists down the package, distro version, Maintainer name etc. which makes them consistent and easier to share with other teams etc. The header also has the script license which we set as "Apache 2.0"

Reply:
Thanks, will do